docs/source/60_building_apps.md

```diff
@@ -77,7 +83,7 @@ Add the new command to the default list of commands in `ADAMMain`.
 
 Build ADAM and run the new command via `adam-submit`.
 
-```bash
+```
```

Comment:
I deleted the `bash` specification here because it caused non-bash text to render with text highlighting. Removing `bash` does not affect anything except the incorrectly highlighted text. In the [rendered version of this PR](https://github.com/devin-petersohn/adam/blob/76913b903e0b256a1f2eb459344ab4ecd4324b8b/docs/source/60_building_apps.md), you can see an example of both `bash` tagged and not.